Handle avg_pool2d with padding == 0 as no padding

3l1 opened the pull request:
Differential Revision: D74117402

3l1 commented:
@pytorchbot label "topic: not user facing"

digantdesai reviewed (approved):
stamping to unblock, left a comment on the diff

3l1 committed:
Handle avg_pool2d with padding == 0 as no padding (#10697)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/10697

Differential Revision: D74117402
